fdbserver/DDShardTracker.actor.cpp

```diff
@@ -1277,14 +1268,6 @@ ACTOR Future<Void> shardEvaluator(DataDistributionTracker* self,
 	bool shouldMerge = stats.bytes < shardBounds.min.bytes && bandwidthStatus == BandwidthStatusLow &&
 	                   (shardForwardMergeFeasible(self, keys, nextIter.range()) ||
 	                    shardBackwardMergeFeasible(self, keys, prevIter.range()));
-	if (onBulkLoading && shouldMerge) {
```

Comment:
Why do we remove this check when the name of this PR is 'Avoid Shard Merge When BulkLoad'? Ditto on the split. Seems like we want to avoid splitting during bulkload?

Reply:
(Or do we want to let it split?)

Reply:
We put the checking outside the shardEvaluator